<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monitor</class>
 <widget class="QDialog" name="monitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>846</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>寿命特征监测模块</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Desktop/src=http___pic.51yuansu.com_pic2_cover_00_33_56_581127cb17246_610.jpg&amp;refer=http___pic.51yuansu.jpg</normaloff>../../Desktop/src=http___pic.51yuansu.com_pic2_cover_00_33_56_581127cb17246_610.jpg&amp;refer=http___pic.51yuansu.jpg</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>失效机理数据管理</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <spacer name="horizontalSpacer_19">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>分类筛选</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_20">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>70</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_18">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>撤销筛选</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_19">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>删除条目</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>70</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>增加行</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_21">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>300</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_20">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>导入</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_21">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>导出</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="4,1">
         <item>
          <widget class="QTableWidget" name="tableWidget_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>2</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>3</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>4</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>5</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>6</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>7</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>8</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>9</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>10</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>11</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>12</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>13</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>14</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>15</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>16</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>17</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>18</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>故障类型</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>失效模式</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>失效原因</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="1" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="2" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="3" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="4" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="5" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="6" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="7" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="8" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="9" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="10" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="11" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="12" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="13" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="14" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="15" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="16" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="17" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_17">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <spacer name="horizontalSpacer_18">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>800</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_17">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>退出</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>元器件参数数据管理</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>分类筛选</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>70</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>撤销筛选</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_9">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>删除条目</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>70</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>增加行</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>300</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>导入</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>导出</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4,1">
         <item>
          <widget class="QTableWidget" name="tableWidget_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>2</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>3</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>4</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>5</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>6</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>7</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>8</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>9</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>10</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>11</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>12</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>13</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>14</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>15</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>16</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>17</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>18</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>所属电路</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>器件名称</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>型号规格</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>封装形式</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>工作温度</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>生产厂商</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>使用数量</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>工作电参数</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>备注</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>用途</string>
            </property>
           </column>
           <item row="0" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="1" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="2" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="3" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="4" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="5" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="6" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="7" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="8" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="9" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="10" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="11" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="12" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="13" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="14" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="15" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="16" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="17" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>800</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>退出</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
